app/transcripts.py

```diff
@@ -0,0 +1,7 @@
+class Transcripts:
+    transcripts=[]
+    summaries=[]
+    def __init__(self, id_, transcripts, summaries):
```

Comment:
Generally, you should stick to using self.x instead of instantiating variables in class body. 
In this situation, you have the 
` transcripts=[]`
and 
`def __init__(self, id_, transcripts, summaries):`
`self.transcripts.append(transcripts)`
If I call Transcripts(id, different_transcripts, different_summaries)
you run into a ghosting problem where you technically have 2 variables named transcripts inside of the init function

Either way, shouldn't break anything, so moving on